### PR TITLE
Implement Deconstructing Damaged Base Plating, also bugfix Scorched Plating to be Repairable

### PR DESCRIPTION
**Pull request by Mute-Man**
### Purpose
 <br>

Implements a way to deconstruct a station's base plating, without using high explosives to make a massive hull breach.
The interaction has a 30 second long progress timer; please let me know if this time should be lengthened or shortened.
Fixes ScorchedPlatings to be repairable, so explosions no longer create permanent marks.

### Notes:

This change is part of my plan to implement the Thermite reaction (Thermite would damage base plating, which would allow for the interaction implemented here.) 

### Changelog:

CL: [Improvement] Changes Base Plating to allow for Deconstruction
CL: [Fix] Scorched Plating can now be Repaired

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -116,4 +116,5 @@ UnityProject/ProjectSettings/Packages/com.xeleh.enhancer/Settings.asset
 UnityProject/AddressablePackingProjects/SoundAndMusic/.vscode/settings.json
 UnityProject/Assets/Resources/ScriptableObjectsSingletons/SpriteCatalogueSingleton.asset
 Tools/CodeScanning/CodeScan/CodeScan/obj
-UnityProject/AddressablePackingProjects/SoundAndMusic/obj
\ No newline at end of file
+UnityProject/AddressablePackingProjects/SoundAndMusic/obj
```
Comment (MaxIsJoe) [inline .gitignore]: why are you editing this?

**Review by MaxIsJoe — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (MaxIsJoe) [reply .gitignore]: just revert it then I'll approve

Comment (Mute-Man) [reply .gitignore]: I, uh… forgor that reverts existed.
Will fix by today, unless git CLI explodes me noggin.

Comment (Mute-Man) [reply .gitignore]: Changes to .gitignore should be reverted now.

**Review by MaxIsJoe — APPROVED** [review]

**Comment by MaxIsJoe** [comment]
Once tests pass I'll merge

**Pull request merged**